pages/common/npx.md

```diff
@@ -3,21 +3,21 @@
 > Execute binaries from `npm` packages.
 > More information: <https://github.com/npm/npx>.
 
-- Execute the binary from a given npm module:
+- Execute the command from a local or remote `npm` package:
 
-`npx {{module_name}} {{command_arguments}}`
+`npx {{command}} {{command_arguments}}`
 
-- In case a package has multiple binaries, specify the package name along with the binary:
+- In case multiple commands with the same name exist, it is possible to specify the package name:
 
-`npx --package {{package_name}} {{module_name}}`
+`npx --package {{package_name}} {{command}}`
 
 - Run a command if it exists in the current path or in `node_modules/.bin`:
 
 `npx --no-install {{command}} {{command_arguments}}`
 
-- Execute the binary from a given npm module suppressing any output from `npx` itself:
+- Execute the command suppressing any output from `npx` itself:
 
-`npx --quiet {{module_name}} {{command_arguments}}`
+`npx --quiet {{command}} {{command_arguments}}`
```

Comment:
```suggestion
`npx --quiet {{command}} {{arg1 arg2 ...}}`
```
Mandatory fix: [ellipsis must be used](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases).